src/workloads/query/TimeseriesTSBSQuery.yml

```diff
@@ -0,0 +1,749 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-integration"
+Description: |
+  This workload runs queries from the tsbs_query benchmark in Genny. It allows us to cross-check the results of the TSBS 
+  benchmarks, collect additional measurements, profile individual queries, and extend some of the queries as needed.
+  The data is preloaded by the dsi configuration. See 'configurations/test_control/test_control.tsbs_query.yml' for all 
+  the details. There are 20736000 documents in the collection.
+  There are some differences between the queries in this workload and tsbs, but we do not expect these differences to significantly
+  affect runtime:
+  1. All TSBS queries have a filter on "measurement": "cpu", which we remove since it is true for all measurements.
+  2. TSBS will generate a random start time between the start/end time of the workload for every query type. We hard code
+  our time values, since genny does not currently support date arithmetic, nor storing a fixed value from randomized generation.
+  3. TSBS randomizes which measurement fields to group by, we do not randomize these fields. Since all the fields are random integers
+  between 0-100 we don't expect different fields to have significant runtime differences.
+
+Keywords:
+  - timeseries
+  - aggregate
+  - group
+  - sort
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 4000 # With 8 threads we need a larger pool.
+
+GlobalDefaults:
+  Database: &database cpu
+  Collection: &collection point_data
+  maxPhases: &maxPhases 20
+  Repeat: &repeat 500
+  # TSBS constructs a time windows of various sizes (1 hour, 8 hours, 12 hours) from a uniformly random start-time within
```

Comment:
The tasks were timing out on evergreen and taking so long to run (over 2 hours), so I'm testing reducing the number of times we repeat each query.